dropshot/tests/fail/bad_trait_endpoint12.stderr

```diff
@@ -1,28 +1,20 @@
-error[E0277]: the trait bound `std::string::String: HttpCodedResponse` is not satisfied
+error[E0277]: endpoint handler functions must return a value that implements `dropshot::HttpResponse`
   --> tests/fail/bad_trait_endpoint12.rs:24:10
    |
 24 |     ) -> Result<String, HttpError>;
-   |          ^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `HttpCodedResponse` is not implemented for `std::string::String`, which is required by `Result<std::string::String, dropshot::HttpError>: dropshot::api_description::HttpResultType`
+   |          ^^^^^^^^^^^^^^^^^^^^^^^^^ the trait `HttpResponse` is not implemented for `Result<std::string::String, dropshot::HttpError>`
    |
-   = help: the following other types implement trait `HttpCodedResponse`:
-             HttpResponseAccepted<T>
-             HttpResponseCreated<T>
-             HttpResponseDeleted
-             HttpResponseOk<T>
-             HttpResponseUpdatedNoContent
```

Comment:
These were really useful suggestions. Is there any way to get these back?

Reply:
They're back as of https://github.com/oxidecomputer/dropshot/pull/1180/commits/00bcea760a2286dbf499deae550986ff2cef8fd1, since we no longer allow endpoints to return `T: HttpResponse` and instead require `Result<T, E> where T: HttpResponse, E: HttpResponseError` again. The error messages are now much closer to how they were before except that the error type can be any `E: HttpResponseError` rather than `HttpError`.